assume_role.sh

```diff
@@ -1,13 +1,15 @@
+#!/bin/bash
+
 if [ "$#" -ne 2 ]
 then
-  echo "Usage: source assume_role.sh [account_id] [role]"
+  echo "Usage: . assume_role.sh [account_id] [role]"
```

Comment:
Can we please change this back to source given we are actually calling it with source again.

Reply:
👍